### DISCUSSION TITLE
handles for scaled objects

### DISCUSSION THREAD

**Discussion by brianp70x** [fabricjs/fabric.js#8056] [General]
Ultimately I want to load object from the SVG so the text can be edited and the return the modified SVG file.

[https://jsfiddle.net/brianp70/xybgpcz2/168/](url)
demonstrates loading the object to the canvas individually, but the when scale is applied the handles stay at the old coords..

[https://jsfiddle.net/uv102sjy/](url)  
loads the group to the canvas and sets scale "correctly", but when I copy objects out of the group. the handles are still misplaced
[https://jsfiddle.net/brianp70/x9gb6ews/5/](url)
select "CW TEXT" and check handle locations

the last example is using v6-aplpha (as suggested previously)
is there a way to make objects within groups selectable on the canvas that I'm missing?
or can I force handles to update?

**Comment by ShaMan123** [comment]
First of all the last example is incorrect.
The cdn points to v5.
Looking into your example I can see you are populating `Text` instances which are not interactive to begin with. You need to do something about that. After you replace the instances with `IText` or `Textbox` this will work https://jsfiddle.net/ep6L91sr/1/

**Reply by brianp70x** [reply]
I greatly appreciate the help. That will work perfectly. I already have the itext to text conversion written, I just wanted to reduce the complications in the fiddle

**Comment by brianp70x** [comment]
any suggestions on how I can improve my search technique? I've spent a couple days looking for "findScaleToFit". when using google "fabric.js scale object to fit canvas" as a search, doesn't do the trick...Thanks again.

**Reply by ShaMan123** [reply]
I feel your pain
Docs are insufficient (being subtle) 
TS will improve things dramatically
V6 stuff are not documented yet
So it's bad

The demos section in the website is a good place to be aware of
Interesting utils reside in the utils/misc file

**Reply by ShaMan123** [reply]
And of course discussions and issues.
Take a look at the tracker issues of Group and TS (they are pinned at the top of issues)
